pkgs/by-name/at/attract-mode/package.nix

```diff
@@ -0,0 +1,86 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  fetchpatch2,
+  pkg-config,
+  expat,
+  ffmpeg_7,
+  freetype,
+  libarchive,
+  libjpeg,
+  libGLU,
+  sfml,
+  zlib,
+  openal,
+  fontconfig,
+  darwin,
+}:
+
+stdenv.mkDerivation {
+  pname = "attract-mode";
+  version = "2.7.0-unstable-2024-06-21";
+
+  src = fetchFromGitHub {
+    owner = "mickelson";
+    repo = "attract";
+    rev = "6ab07d2e23a447b8cf56e927a3fe707322820024";
+    hash = "sha256-1RmwNebFUvnbUtPNxTAFlnZK/2DCmJWAYZtP1y9xepU=";
+  };
+
+  patches = [
+    # Build Objective-C++ files with the C++ compiler
+    # https://github.com/mickelson/attract/pull/757
```

Comment:
Now merged; can drop patch and just update to that revision.

Reply:
Totally missed that getting merged! Pushed again, thanks.